library/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneFlightTest.java

```diff
@@ -58,22 +44,23 @@ void testInitializationWhenAttaching() {
     createLandingZoneFlight = new CreateLandingZoneFlight(inputParameters, mockApplicationContext);
 
     var steps = createLandingZoneFlight.getSteps();
-    assertThat(steps.size(), equalTo(3));
+    assertThat(steps.size(), equalTo(2));
     validateSteps(steps, isAttaching);
   }
 
   void validateSteps(List<Step> steps, boolean isAttaching) {
     assertThat(steps.stream().filter(s -> s instanceof CreateSamResourceStep).count(), equalTo(1L));
-    assertThat(steps.stream().filter(s -> s instanceof GetBillingProfileStep).count(), equalTo(1L));
     if (!isAttaching) {
-      assertThat(
-          steps.stream().filter(s -> s instanceof CreateLandingZoneResourcesFlightStep).count(),
-          equalTo(1L));
+      // don't want to test for the exact number because it's perfectly valid that it can change
+      // over time,
+      // and this isn't a test for the step factory
```

Comment:
How are the step factories tested?  I looked for unit tests but couldn't find any under the obvious names.

Reply:
I had a fairly lengthy reply to a comment you made on `TestCreateLandingZoneResourcesFlight` that I think addressed this, but I don't see either the original comment, or the reply anymore....

The general idea of it, was that `CreateLandingZoneResourcesFlightIntegrationTest` is basically using `TestCreateLandingZoneResourcesFlight` to isolate the resource steps.